contents/handbook/company/goal-setting.md

```diff
@@ -49,7 +50,7 @@ Bear the following in mind:
 - Use counter metrics where needed (X happens, but Y shouldn't happen)
 - Don't have a lot of Key Results if you can't capture everything in one - just pick the most important one or two
 - Don't set arbitrary Key Results that a team cannot achieve
-- Don't set a Key Result that is mainly noise or a vanity metric
+- Don't set a Key Result that is mainly noise or a vanity metric, and try to avoid boolean (yes/no) Key Results
```

Comment:
I'd propose not adding this. Here's some context (which was counterintuitive for me at least)...

One thing that has stood out, and I think it's due to being very product led and with much of the platform being very early, is that yes/no "ship X" key results I think _has_ led to greater performance than numerical key results. We've seen it last quarter with the style of objective for team east versus the others

I even had a discussion on this topic with the founder of Sentry (much larger than us) and they've experienced the same thing. I think it's because we're neither Google scale _nor_ sales led that this works well - a lot of objectives are quite qualitative (we do the right thing, our brand improves, we get more adoption and growth longer term, but not in a measurable way _every time_ (although yes sometimes!) each quarter).

Curious what your reaction to the above is?

Reply:
I think my rationale for including something like this is that what sets the OKR process apart from other systems for me is its focus on the outputs of certain activities rather than the activities themselves. I've found that being able to quantitatively measure _what_ a given objective is achieving for us allows us to not only track what we are getting done but to also use a metricized retroactive reflection process to progressively get _better_ at goal setting and building as we go.

Measurable key results allow us to understand whether we are setting our bar too low or too aggressively (IE always scoring a 1, or never making it to the ideal 0.7-0.9 target, would tell us that our OKR setting process needs to be fine-tuned) and whether what we are working on is actually having the impact that we hypothesize it should have. I find that yes/no or checkbox-style key results don't provide that kind of clarity, and don't push us to think really critically about whether we are prioritizing the most important work in terms of tangible outputs from shipping projects.

I also think there's no harm in being a bit more flexible in how we write our KRs this early on in adopting the process because after all the whole system is about experimentation and dynamic resource allocation.

Reply:
ok, I think "try to avoid" then is reasonable language here.